apps/dashboard/app/controllers/products_controller.rb

```diff
@@ -115,6 +115,7 @@ def cli
     response.headers['Content-Type'] = 'text/plain'
     Dir.chdir(@product.router.path) do
       Bundler.with_clean_env do
+        ENV['BUNDLE_USER_CONFIG'] = '/dev/null' if Rails.env.production?
```

Comment:
Do these ENV variables propagate to the sub shell? I think we may need to pass them explicitly to the popen2e api.

Reply:
For backticks, yes. I didn't test the other changes, just the one executing `setup-production`.

Reply:
I would presume yes because we're using `Bundler.with_clear_env` unless that's a useless block in some cases.

Reply:
Yea sorry, they do, I quickly tested in irb and they do propogate to open3 functions.